assets/terraform/libvirt/output.tf

```diff
@@ -0,0 +1,11 @@
+#
+# Output Variables
+#
+
+output "private_ips" {
+  value = "${libvirt_domain.domain-gravity.*.network_interface.0.addresses.0}"
+}
+
+output "public_ips" {
+  value = "${libvirt_domain.domain-gravity.*.network_interface.0.addresses.0}"
```

Comment:
This seems like it will output exactly the same as "private_ips" above.

Reply:
Yeah, I'm still not entirely sure what the public IPs are used for, but they are a required output.
Any advice on where to look to get a better understanding of how the public IPs are used in gravity?

Reply:
Public IPs have been historically used to connect to nodes via SSH to run remote commands.